config/application.rb

```diff
@@ -22,7 +22,9 @@ class Application < Rails::Application
 
     Rails.application.routes.default_url_options[:protocol] = 'https'
     config.log_formatter = ::ActiveSupport::Logger::SimpleFormatter.new
-    
+
+    config.pdftk = PdfForms.new(Rails.root.join('bin/pdftk').to_s)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.